workers/unity/Assets/Gdk/Physics/Systems/ApplyTransformUpdatesSystem.cs

```diff
@@ -10,7 +11,8 @@ public class ApplyTransformUpdatesSystem : ComponentSystem
         public struct TransformUpdateData
         {
             public int Length;
-            public ComponentArray<ComponentsUpdated<SpatialOSTransform>> TransformUpdate;
+            public ComponentDataArray<SpatialOSTransform> Transform;
```

Comment:
we are not using this one in the system. Is it needed?

Reply:
We are using this in line 47. :)